package.json

```diff
@@ -90,26 +90,26 @@
     "eslint-plugin-rulesdir": "^0.1.0",
     "eslint-release": "^1.2.0",
     "eslint-rule-composer": "^0.3.0",
-    "eslump": "^1.6.2",
+    "eslump": "^2.0.0",
     "esprima": "^4.0.1",
     "istanbul": "^0.4.5",
     "jsdoc": "^3.5.5",
-    "karma": "^3.0.0",
-    "karma-babel-preprocessor": "^7.0.0",
+    "karma": "^3.1.4",
+    "karma-babel-preprocessor": "^8.0.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.3",
     "leche": "^2.2.3",
     "load-perf": "^0.2.0",
-    "markdownlint": "^0.11.0",
+    "markdownlint": "^0.12.0",
     "mocha": "^5.0.5",
-    "mock-fs": "^4.6.0",
+    "mock-fs": "^4.8.0",
```

Comment:
It's not a big deal, but I think this `mock-fs` upgrade should have happened at the same time as 1309dfdebb5595460b79dcac20df6a1f109e7566. The build on master was failing for me locally because I still had `mock-fs@4.6.0` installed.

Reply:
it should be.  I thought this PR could be merged before that one.